docs/user-guide/commands/argocd_app_delete.md

```diff
@@ -14,6 +14,7 @@ argocd app delete APPNAME [flags]
 
 ```
       --cascade   Perform a cascaded deletion of all application resources (default true)
+      --force     Turn off prompting to confirm cascaded deletion of application resources
```

Comment:
There was some discussion in the previous PR #4939 about maybe using something other than `--force` for the flag.  Maybe `--noPrompt` or `-y` would be more appropriate?

Reply:
I made the UI changes so far only.  I will make the CLI-related changes based on everyone's feedback.